lib/cutest.rb

```diff
@@ -148,19 +148,19 @@ def test(name = nil, &block)
     cutest[:test] = nil
   end
 
-  # Assert that value is not nil or false.
-  def assert(value)
-    flunk("expression returned #{value.inspect}") unless value
+  # Assert that test is a true value.
+  def assert(test)
+    flunk("expression returned #{test.inspect}") unless test
     success
   end
 
-  # Assert that two values are equal.
-  def assert_equal(value, other)
-    flunk("#{value.inspect} != #{other.inspect}") unless value == other
+  # Assert that expected and actual values are equal.
+  def assert_equal(expected, actual)
```

Comment:
@xymbol Yes, I think we should leave the parameters as they are. It feels more natural to me, it's clearer that `#==` is being called on the first argument, etc. That is, unless there are compelling reasons to change it. (My guess is Minitest uses the order Test::Unit used, which in turn was some port from a Java project? I don't remember.)